# ETL Project: Leading causes of death in the United States

#### Aim:
An analysis on the top 10 leading causes of death in the United States. This case study seeks to analyze the top causes of deaths from 2013 to 2016 by ranking death causes by year, state, and age-adjusted deaths. Further, would U.S. chronic disease indicators provide any insight into risk factors that may be associated with those leading causes of death? 

#### Notes:
* The 10 leading causes of death are classified by the International Classification of Diseases, Tenth Revision (ICD-10).
* Age-adjusted death rates are per 100,000 standard million population in the year 2000. 


#### Data sources:
* [CDC/HCHS](https://catalog.data.gov/dataset/age-adjusted-death-rates-for-the-top-10-leading-causes-of-death-united-states-2013)

* [CDC](https://catalog.data.gov/dataset/u-s-chronic-disease-indicators-cdi)

#### Tools:
* Jupter Notebook/Python 3
* MySQL

# Leading Causes Of Death In The United States
## Data Extraction

After downloading csv files from our sources, csv files were extracted into dataframes utilizing jupyter notebook python 3 pandas function pd.read_csv().

In [1]:
# import dependencies
import pandas as pd
import pymysql
from sqlalchemy import create_engine

In [2]:
# read csv file for data on leading causes of death
leading_causes_death_df = pd.read_csv('Resources/NCHS_-_Leading_Causes_of_Death__United_States.csv')

In [3]:
# preview the dataframe created
leading_causes_death_df.head()

Year                                     113 Cause Name  \
0  2016  Accidents (unintentional injuries) (V01-X59,Y8...   
1  2016  Accidents (unintentional injuries) (V01-X59,Y8...   
2  2016  Accidents (unintentional injuries) (V01-X59,Y8...   
3  2016  Accidents (unintentional injuries) (V01-X59,Y8...   
4  2016  Accidents (unintentional injuries) (V01-X59,Y8...   

               Cause Name       State  Deaths  Age-adjusted Death Rate  
0  Unintentional injuries     Alabama    2755                     55.5  
1  Unintentional injuries      Alaska     439                     63.1  
2  Unintentional injuries     Arizona    4010                     54.2  
3  Unintentional injuries    Arkansas    1604                     51.8  
4  Unintentional injuries  California   13213                     32.0

## Data Transformation
Due to the specificity contained in the column '113 Cause Name' in the leading_causes_death dataframe, it was decided that broader categories of death causes should be used to more closely match the Chronic Disease Indicators' categories for easier grouping of the two datasets. Thus utilizing the column 'Cause Name' as our main category. 

The column 'Age-adjusted Death Rate' was removed due to the lack of clarity from the datasource and not being able to to rank leading causes of death in a meaningful way. Columns were also renamed for easier analysis through python 3.

In [4]:
## cleaning the dataframe

# Remove cause_name and state columns, Rename columns into a new dataframe
leading_causes = leading_causes_death_df[['Year', 'Cause Name', 'State','Deaths']].copy()
leading_causes = leading_causes.rename(columns={
    "Year":"year",
    "Cause Name": "cause_name",
    "State": "state",
    "Deaths": "deaths"
})
leading_causes.sort_values(by=["year"]).head()

year              cause_name          state  deaths
10295  1999  Unintentional injuries        Wyoming     258
1418   1999     Alzheimer's disease      Minnesota    1083
8726   1999                 Suicide       Illinois    1020
7016   1999          Kidney disease       Michigan    1417
4334   1999                Diabetes  New Hampshire     294

Since the data sets end in 2016, we chose the time frame from 2013 to 2016. This three year period was to sample for any possible relationships between leading causes of death and chronic disease indicators.


In [5]:
# Filter leading_causes for years >=2013
leading_causes_2013_to_2016 = leading_causes.loc[leading_causes['year'] >= 2013,:]
leading_causes_2013_to_2016=leading_causes_2013_to_2016.sort_values(by=['year'])
leading_causes_2013_to_2016.head()

year              cause_name          state  deaths
10281  2013  Unintentional injuries        Wyoming     325
3708   2013                    CLRD       Virginia    3181
3726   2013                    CLRD     Washington    2933
8946   2013                 Suicide        Montana     243
3744   2013                    CLRD  West Virginia    1590

One of the categories in "cause_name" is "All causes". Since this is a summation of all the categories in "cause_name", it was decided that the "All causes" category should be removed from the dataset.

In [6]:
# Remove all causes from dataframe
leading_causes_2013_to_2016_individual = leading_causes_2013_to_2016[leading_causes_2013_to_2016.cause_name != "All causes"]
leading_causes_2013_to_2016_individual.head()

year              cause_name          state  deaths
10281  2013  Unintentional injuries        Wyoming     325
3708   2013                    CLRD       Virginia    3181
3726   2013                    CLRD     Washington    2933
8946   2013                 Suicide        Montana     243
3744   2013                    CLRD  West Virginia    1590

Since the data set merged both all of the United States and individual states, we separated the two areas to take into account of the leading causes of death in the United States and in each individual state by year.

In [ ]:
# Separate US from individual states (2013 to 2016)
leading_causes_2013_to_2016_individual_US = leading_causes_2013_to_2016_individual[leading_causes_2013_to_2016_individual.state == "United States"]
leading_causes_2013_to_2016_individual_US.head()

In [ ]:
# ALL US 2013 Top 10 leading deaths
# Isolate only 2013 data
top_10_leading_2013_US = leading_causes_2013_to_2016_individual_US.loc[leading_causes_2013_to_2016_individual_US['year']==2013,:]
top_10_leading_2013_US = top_10_leading_2013_US.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2013_US

In [ ]:
# ALL US 2014 Top 10 leading deaths
# Isolate only 2014 data
top_10_leading_2014_US = leading_causes_2013_to_2016_individual_US.loc[leading_causes_2013_to_2016_individual_US['year']==2014,:]
top_10_leading_2014_US = top_10_leading_2014_US.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2014_US

In [ ]:
# ALL US 2015 Top 10 leading deaths
# Isolate only 2015 data
top_10_leading_2015_US = leading_causes_2013_to_2016_individual_US.loc[leading_causes_2013_to_2016_individual_US['year']==2015,:]
top_10_leading_2015_US = top_10_leading_2015_US.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2015_US

In [ ]:
# ALL US 2016 Top 10 leading deaths
# Isolate only 2016 data
top_10_leading_2016_US = leading_causes_2013_to_2016_individual_US.loc[leading_causes_2013_to_2016_individual_US['year']==2016,:]
top_10_leading_2016_US = top_10_leading_2016_US.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2016_US

In [ ]:
# Separate out states from 2013 to 2016
leading_causes_2013_to_2016_individual_states = leading_causes_2013_to_2016_individual[leading_causes_2013_to_2016_individual.state != "United States"]
leading_causes_2013_to_2016_individual_states.head()

In [ ]:
# ALL STATES 2013 Top 10 leading deaths and which states had the most deaths in the year 2013
# Isolate only 2013 data
leading_2013_states = leading_causes_2013_to_2016_individual_states.loc[leading_causes_2013_to_2016_individual_states['year']==2013,:]
top_10_leading_2013_states_all = leading_2013_states.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2013_states_all

In [ ]:
# Top 10 unique states 2013
# Sort data by number of death from highest to lowest, drop any duplicate states for this year, and obtain only the top 10 states
top_10_leading_2013_states_unique = leading_2013_states.sort_values(by=['deaths'], ascending=False).drop_duplicates(subset='state', keep='first').head(10)
top_10_leading_2013_states_unique

In [ ]:
# ALL STATES 2014 Top 10 leading deaths and which states had the most deaths in the year 2014
# Isolate only 2014 data
leading_2014_states = leading_causes_2013_to_2016_individual_states.loc[leading_causes_2013_to_2016_individual_states['year']==2014,:]
top_10_leading_2014_states_all = leading_2014_states.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2014_states_all

In [ ]:
# Top 10 unique states 2014
# Sort data by number of death from highest to lowest, drop any duplicate states for this year, and obtain only the top 10 states
top_10_leading_2014_states_unique = leading_2014_states.sort_values(by=['deaths'], ascending=False).drop_duplicates(subset='state', keep='first').head(10)
top_10_leading_2014_states_unique

In [ ]:
# ALL STATES 2015 Top 10 leading deaths and which states had the most deaths in the year 2015
# Isolate only 2015 data
leading_2015_states = leading_causes_2013_to_2016_individual_states.loc[leading_causes_2013_to_2016_individual_states['year']==2015,:]
top_10_leading_2015_states_all = leading_2015_states.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2015_states_all

In [ ]:
# Top 10 unique states 2015
# Sort data by number of death from highest to lowest, drop any duplicate states for this year, and obtain only the top 10 states
top_10_leading_2015_states_unique = leading_2015_states.sort_values(by=['deaths'], ascending=False).drop_duplicates(subset='state', keep='first').head(10)
top_10_leading_2015_states_unique

In [ ]:
# ALL STATES 2016 Top 10 leading deaths and which states had the most deaths in the year 2016
# Isolate only 2016 data
leading_2016_states = leading_causes_2013_to_2016_individual_states.loc[leading_causes_2013_to_2016_individual_states['year']==2016,:]
top_10_leading_2016_states_all = leading_2016_states.sort_values(by=['deaths'], ascending=False).head(10)
top_10_leading_2016_states_all

In [ ]:
# Top 10 unique states 2016
# Sort data by number of death from highest to lowest, drop any duplicate states for this year, and obtain only the top 10 states
top_10_leading_2016_states_unique = leading_2016_states.sort_values(by=['deaths'], ascending=False).drop_duplicates(subset='state', keep='first').head(10)
top_10_leading_2016_states_unique

# Chronic Disease Indicators

#### Column Descriptions:
* YearStart - Starting Year
* YearEnd - Ending Year
* LocationAbbr - Location Abbreviation
* LocationDesc - Location Description
* DataSource - Data Source Abbreviation
* Topic - Topic
* Question - Question full-length text
* Response - Response
* DataValueUnit - The unit, such as $, %, years, etc.
* DataValueType - The data type, such as prevalence or mean
* DataValue - Data Value, such as 14.7 or Category 1
* DataValueAlt - Equal to Data Value, but formatting is numeric
* DataValueFootnoteSymbol	- Footnote Symbol
* DatavalueFootnote - Footnote Text
* LowConfidenceLimit - Low Confidence Limit
* HighConfidenceLimit - High Confidence Limit
* StratificationCategory1	- The category of the stratification, such as * Gender, Overall, or Race/Ethnicity
* Stratification1	- The stratification within the category, such as Male or Female, White/non-Hispanic, Hispanic, Black/non-Hispanic, American Indian or Alaska Native, Asian or Pacific Islander, Multi-racial/non-Hispanic, Other/non-Hispanic, or Overall
* StratificationCategory2
* Stratification2	
* StratificationCategory3
* Stratification3	
* GeoLocation	- Location code to be used for Geocoding
* ResponseID - Identifier for the Response
* LocationID - Location Identifier
* TopicID	
* QuestionID - Question Identifier
* DataValueTypeID	- Identifier for the Data Value Type
* StratificationCategoryID1 - Identifier for stratification category 1
* StratificationID1 - Identifier for stratification 1
* StratificationCategoryID2	
* StratificationID2	
* StratificationCategoryID3	
* StratificationID3	


In [ ]:
# read chronic diseases indicators csv
CDI_df = pd.read_csv('Resources/U.S._Chronic_Disease_Indicators__CDI_.csv')
CDI_df.shape

This dataset contained several columns of missing or not relevant information to the analysis. 

The following steps below:
* removed unnecessary columns that contained missing or repetitive information
* separated datasets by location of either 'United States' or individual states
* categorized two larger datasets by either 'Crude Prevalence' or 'Age-adjusted Prevalence', which was then stratified by either overall, male, or female gender types ('Stratification1' column). 
* separated each gender type by year (2013, 2014, 2015, 2016)

In [ ]:
## CLEAN UP DATAFRAME
# Select desired columns
CDI_df = CDI_df[['YearStart', 'LocationAbbr', 'LocationDesc', 'Topic', 'DataValue', 'DataValueUnit', 'DataValueType', 'StratificationCategory1', 'Stratification1']]

CDI_df.shape

In [ ]:
# Drop any other rows with missing data
CDI_df = CDI_df.dropna(subset = ['DataValue', 'DataValueUnit'])
CDI_df.shape

In [ ]:
# Remove rows where 'DataValue' == ' '
CDI_df = CDI_df.loc[CDI_df['DataValue'] != ' ']

# # Remove texts from dataValue
# CDI_df = CDI_df.loc[CDI_df['DataValueUnit'] != ""]


# # # Remove the string "No"
# # CDI_df = CDI_df.loc[CDI_df['DataValue'] != ("No" or\
# #                                             "Yes" or\
# #                                             "Category 2 - State had commercial host liability with major limitations" or\
# #                                             "Category 1 - State had commercial host liability with no major limitations" or\
# #                                             "Category 3 - State had no commercial host liability" or\
# #                                             "Category 3 - State had exclusive state alcohol retail licensing but with local zoning authority" or\
# #                                             "Category 4 - State had mixed alcohol retail licensing policies\xa0" or\
# #                                             "Category 2 - State had joint local and state alcohol retail licensing" or\
# #                                             "Category 6 - State had exclusive state alcohol retail licensing" or\
# #                                             "Category 1 - State had exclusive local alcohol retail licensing" or\
# #                                             "Category 5 - State had nearly exclusive state alcohol retail licensing" or\
# #                                             "Category 4 - State had mixed alcohol retail licensing policies"),:]
        
    
    
# CDI_df.loc[CDI_df['DataValue']=="Category 2 - State had commercial host liability with major limitations"]


In [ ]:
# Convert DataValue from string to float

CDI_df.DataValue = [float(x) for x in CDI_df.DataValue]
CDI_df.head()


## Chronic Disease Indicators by United States

In [ ]:
#### PLACE HOLDER

## Chronic Disease Indicators Analysis By State

In [ ]:
# Obtain rows only for the all states
CDI_states = CDI_df.loc[CDI_df['LocationAbbr'] !="US",:]

# Obtain rows only for the years >=2013
CDI_states = CDI_states.loc[CDI_states['YearStart'] >=2013,:]
CDI_states.head(10)

In [ ]:
# Separate crude prevalence and age-adjusted prevalence into separate dataframes
CDI_states_crude = CDI_states.loc[CDI_states['DataValueType']=="Crude Prevalence",:]
CDI_states_age_adjusted = CDI_states.loc[CDI_states['DataValueType']=="Age-adjusted Prevalence", :]


In [ ]:
# Separate CDI_states_crude into 3 stratification categories: Overall, Gender Male, Gender Female

# Overall
CDI_states_crude_overall = CDI_states_crude.loc[CDI_states_crude['Stratification1']=="Overall",:]
CDI_states_crude_overall = CDI_states_crude_overall.sort_values(by=['DataValue'], ascending=False)
# Male
CDI_states_crude_male = CDI_states_crude.loc[CDI_states_crude['Stratification1']=="Male",:]
CDI_states_crude_male = CDI_states_crude_male.sort_values(by=['DataValue'], ascending=False)
# Female
CDI_states_crude_female = CDI_states_crude.loc[CDI_states_crude['Stratification1']=="Female",:]
CDI_states_crude_female = CDI_states_crude_female.sort_values(by=['DataValue'], ascending=False)

In [ ]:
## Separate CDI_states_crude_{overall, male, female} by year

# CDI state crude overall (sco)
CDI_sco_2013 = CDI_states_crude_overall.loc[CDI_states_crude_overall['YearStart']==2013,:]
CDI_sco_2014 = CDI_states_crude_overall.loc[CDI_states_crude_overall['YearStart']==2014,:]
CDI_sco_2015 = CDI_states_crude_overall.loc[CDI_states_crude_overall['YearStart']==2015,:]
CDI_sco_2016 = CDI_states_crude_overall.loc[CDI_states_crude_overall['YearStart']==2016,:]

# CDI state crude male (scm)
CDI_scm_2013 = CDI_states_crude_male.loc[CDI_states_crude_male['YearStart']==2013,:]
CDI_scm_2014 = CDI_states_crude_male.loc[CDI_states_crude_male['YearStart']==2014,:]
CDI_scm_2015 = CDI_states_crude_male.loc[CDI_states_crude_male['YearStart']==2015,:]
CDI_scm_2016 = CDI_states_crude_male.loc[CDI_states_crude_male['YearStart']==2016,:]

# CDI state crude female (scf)
CDI_scf_2013 = CDI_states_crude_female.loc[CDI_states_crude_female['YearStart']==2013,:]
CDI_scf_2014 = CDI_states_crude_female.loc[CDI_states_crude_female['YearStart']==2014,:]
CDI_scf_2015 = CDI_states_crude_female.loc[CDI_states_crude_female['YearStart']==2015,:]
CDI_scf_2016 = CDI_states_crude_female.loc[CDI_states_crude_female['YearStart']==2016,:]

In [ ]:
# Separate CDI_states_age_adjusted into 3 stratification categories: Overall, Gender Male, Gender Female, 
# sort by descending prevalence % ('DataValue' column)

# Overall
CDI_states_age_adjusted_overall = CDI_states_age_adjusted.loc[CDI_states_age_adjusted['Stratification1']=="Overall",:]
CDI_states_age_adjusted_overall = CDI_states_age_adjusted_overall.sort_values(by=['DataValue'], ascending=False)
# Male
CDI_states_age_adjusted_male = CDI_states_age_adjusted.loc[CDI_states_age_adjusted['Stratification1']=="Male",:]
CDI_states_age_adjusted_male = CDI_states_age_adjusted_male.sort_values(by=['DataValue'], ascending=False)
# Female
CDI_states_age_adjusted_female = CDI_states_age_adjusted.loc[CDI_states_age_adjusted['Stratification1']=="Female",:]
CDI_states_age_adjusted_female = CDI_states_age_adjusted_female.sort_values(by=['DataValue'], ascending=False)

In [ ]:
## Separate CDI_states_age_adjusted_{overall, male, female} by year

# CDI state crude overall (sco)
CDI_sao_2013 = CDI_states_age_adjusted_overall.loc[CDI_states_age_adjusted_overall['YearStart']==2013,:]
CDI_sao_2014 = CDI_states_age_adjusted_overall.loc[CDI_states_age_adjusted_overall['YearStart']==2014,:]
CDI_sao_2015 = CDI_states_age_adjusted_overall.loc[CDI_states_age_adjusted_overall['YearStart']==2015,:]
CDI_sao_2016 = CDI_states_age_adjusted_overall.loc[CDI_states_age_adjusted_overall['YearStart']==2016,:]

# CDI state crude male (scm)
CDI_sam_2013 = CDI_states_age_adjusted_male.loc[CDI_states_age_adjusted_male['YearStart']==2013,:]
CDI_sam_2014 = CDI_states_age_adjusted_male.loc[CDI_states_age_adjusted_male['YearStart']==2014,:]
CDI_sam_2015 = CDI_states_age_adjusted_male.loc[CDI_states_age_adjusted_male['YearStart']==2015,:]
CDI_sam_2016 = CDI_states_age_adjusted_male.loc[CDI_states_age_adjusted_male['YearStart']==2016,:]

# CDI state crude female (scf)
CDI_saf_2013 = CDI_states_age_adjusted_female.loc[CDI_states_age_adjusted_female['YearStart']==2013,:]
CDI_saf_2014 = CDI_states_age_adjusted_female.loc[CDI_states_age_adjusted_female['YearStart']==2014,:]
CDI_saf_2015 = CDI_states_age_adjusted_female.loc[CDI_states_age_adjusted_female['YearStart']==2015,:]
CDI_saf_2016 = CDI_states_age_adjusted_female.loc[CDI_states_age_adjusted_female['YearStart']==2016,:]

## Data Loading into MySQL

In [ ]:
engine = create_engine("mysql://root:Codingislife92!@localhost:3306/leading_causes_of_death_db")

In [ ]:
# Load causes by unique states from 2013 to 2016 data
top_10_leading_2013_states_unique.to_sql(name='top_10_2013_unique_states', con=engine, if_exists='replace', index=True)
top_10_leading_2014_states_unique.to_sql(name='top_10_2014_unique_states', con=engine, if_exists='replace', index=True)
top_10_leading_2015_states_unique.to_sql(name='top_10_2015_unique_states', con=engine, if_exists='replace', index=True)
top_10_leading_2016_states_unique.to_sql(name='top_10_2016_unique_states', con=engine, if_exists='replace', index=True)


In [ ]:
# Load states crude overall (sco) from 2013 to 2016 data
CDI_sco_2013.to_sql(name='cdi_sco_2013', con=engine, if_exists='replace', index=True)
CDI_sco_2014.to_sql(name='cdi_sco_2014', con=engine, if_exists='replace', index=True)
CDI_sco_2015.to_sql(name='cdi_sco_2015', con=engine, if_exists='replace', index=True)
CDI_sco_2016.to_sql(name='cdi_sco_2016', con=engine, if_exists='replace', index=True)

In [ ]:
# Load states crude male (scm) from 2013 to 2016 data
CDI_scm_2013.to_sql(name='cdi_scm_2013', con=engine, if_exists='replace', index=True)
CDI_scm_2014.to_sql(name='cdi_scm_2014', con=engine, if_exists='replace', index=True)
CDI_scm_2015.to_sql(name='cdi_scm_2015', con=engine, if_exists='replace', index=True)
CDI_scm_2016.to_sql(name='cdi_scm_2016', con=engine, if_exists='replace', index=True)


In [ ]:
# Load states crude female (scf) from 2013 to 2016 data
CDI_scf_2013.to_sql(name='cdi_scf_2013', con=engine, if_exists='replace', index=True)
CDI_scf_2014.to_sql(name='cdi_scf_2014', con=engine, if_exists='replace', index=True)
CDI_scf_2015.to_sql(name='cdi_scf_2015', con=engine, if_exists='replace', index=True)
CDI_scf_2016.to_sql(name='cdi_scf_2016', con=engine, if_exists='replace', index=True)


In [ ]:
# Load states age-adjusted overall (sao) from 2013 to 2016 data
CDI_sao_2013.to_sql(name='cdi_sao_2013', con=engine, if_exists='replace', index=True)
CDI_sao_2014.to_sql(name='cdi_sao_2014', con=engine, if_exists='replace', index=True)
CDI_sao_2015.to_sql(name='cdi_sao_2015', con=engine, if_exists='replace', index=True)
CDI_sao_2016.to_sql(name='cdi_sao_2016', con=engine, if_exists='replace', index=True)


In [ ]:
# Load states age-adjusted male (sam) from 2013 to 2016 data
CDI_sam_2013.to_sql(name='cdi_sam_2013', con=engine, if_exists='replace', index=True)
CDI_sam_2014.to_sql(name='cdi_sam_2014', con=engine, if_exists='replace', index=True)
CDI_sam_2015.to_sql(name='cdi_sam_2015', con=engine, if_exists='replace', index=True)
CDI_sam_2016.to_sql(name='cdi_sam_2016', con=engine, if_exists='replace', index=True)


In [ ]:
# Load states age-adjusted female (saf) from 2013 to 2016 data
CDI_saf_2013.to_sql(name='cdi_saf_2013', con=engine, if_exists='replace', index=True)
CDI_saf_2014.to_sql(name='cdi_saf_2014', con=engine, if_exists='replace', index=True)
CDI_saf_2015.to_sql(name='cdi_saf_2015', con=engine, if_exists='replace', index=True)
CDI_saf_2016.to_sql(name='cdi_saf_2016', con=engine, if_exists='replace', index=True)


## Export files to CSV for backup

In [ ]:
# Setup path
p

In [ ]:
# Export to csv causes by unique states from 2013 to 2016 data
top_10_leading_2013_states_unique.to_csv("top_10_leading_causes_2013_states_unique.csv")
top_10_leading_2014_states_unique.to_csv("top_10_leading_causes_2014_states_unique.csv")
top_10_leading_2015_states_unique.to_csv("top_10_leading_causes_2015_states_unique.csv")
top_10_leading_2016_states_unique.to_csv("top_10_leading_causes_2016_states_unique.csv")

# Export to csv states crude overall (sco) from 2013 to 2016 data
CDI_sco_2013.to_csv("CDI_sco_2013.csv")
CDI_sco_2014.to_csv("CDI_sco_2014.csv")
CDI_sco_2015.to_csv("CDI_sco_2015.csv")
CDI_sco_2016.to_csv("CDI_sco_2016.csv")

# Export to csv states crude male (scm) from 2013 to 2016 data
CDI_scm_2013.to_csv("CDI_scm_2013.csv")
CDI_scm_2014.to_csv("CDI_scm_2014.csv")
CDI_scm_2015.to_csv("CDI_scm_2015.csv")
CDI_scm_2016.to_csv("CDI_scm_2016.csv")

# Export to csv states crude female (scf) from 2013 to 2016 data
CDI_scf_2013.to_csv("CDI_scf_2013.csv")
CDI_scf_2014.to_csv("CDI_scf_2014.csv")
CDI_scf_2015.to_csv("CDI_scf_2015.csv")
CDI_scf_2016.to_csv("CDI_scf_2016.csv")

# Export to csv states age-adjusted overall (sao) from 2013 to 2016 data
CDI_sao_2013.to_csv("CDI_sao_2013.csv")
CDI_sao_2014.to_csv("CDI_sao_2014.csv")
CDI_sao_2015.to_csv("CDI_sao_2015.csv")
CDI_sao_2016.to_csv("CDI_sao_2016.csv")

# Export to csv  states age-adjusted male (sam) from 2013 to 2016 data
CDI_sam_2013.to_csv("CDI_sam_2013.csv")
CDI_sam_2014.to_csv("CDI_sam_2014.csv")
CDI_sam_2015.to_csv("CDI_sam_2015.csv")
CDI_sam_2016.to_csv("CDI_sam_2016.csv")

# Export to csv states age-adjusted female (sam) 2013 to 2016 data
CDI_saf_2013.to_csv("CDI_saf_2013.csv")
CDI_saf_2014.to_csv("CDI_saf_2014.csv")
CDI_saf_2015.to_csv("CDI_saf_2015.csv")
CDI_saf_2016.to_csv("CDI_saf_2016.csv")

